utils/io/fileutils/files.go

```diff
@@ -404,6 +410,15 @@ func CopyDir(fromPath, toPath string, includeDirs bool) error {
 	return err
 }
 
+func IsStringInSlice(string string, strings []string) bool {
```

Comment:
I suspsect we can get a nil pointer here.
The *strings* argument can be nil, because we do sent nil to the CopyDir function here - ```err := CopyDir(v, toPath, true, nil)```

Reply:
range on nil array in go behaves the same as range on empty array